Feature: action button (#55)
lightelligence-io/styles: src/components/ActionButton/ActionButton.md

---
name: ActionButton
category: Component
---

## Description

V2-Buttons can be used with `<button>` and `<a>` elements and occupy the `olt-ActionButton*` class names.

> The Action Button consists of three three state, depending on the cursor's distance to the click-area:

Reviewer: ```suggestion
The Action Button consists of three states, depending on the cursor's distance to the click-area:
```

            </div>
            <div class="demo-content">
              <button
                class="olt-ActionButton olt-ActionButton--confirmative olt-ActionButton-icon-left olt-Icon-action-refresh olt-ActionButton--standalone">
                  <div class="olt-ActionButton-action-label">
                    Label
                  </div>
                </button>
            </div>
            <div class="demo-content">
              <button
                class="olt-ActionButton olt-ActionButton--confirmative olt-ActionButton-icon-left olt-Icon-action-refresh olt-ActionButton--standalone olt-ActionButton--base">
                  <div class="olt-ActionButton-action-label">
                    Label
                  </div>
                </button>
            </div>
            <div class="demo-content">
              <button
                class="olt-ActionButton olt-ActionButton--confirmative olt-ActionButton-icon-right olt-Icon-action-refresh">
                <div class="olt-ActionButton-action-label">
                  Label
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>

> On Tablet and Mobile Devices the action button is always displayed as the "Standalone with Base" variant to increase and show the click-area. If the standalone version cannot satisfactory represent the action it can be assisted with the label ("Fixed Label") depending on the context

Reviewer: ```suggestion
On Tablet and Mobile Devices the action button is always displayed as the "Standalone with Base" variant to increase and show the click-area. If the standalone version cannot properly represent the action it should be used together with the label ("Fixed Label") depending on the context.
```
